docs/classes/DocGenerator.md

```diff
@@ -1,577 +1,173 @@
-<embed> <a href="/docs/README.md">BumbleDocGen</a> <b>/</b> DocGenerator<hr> </embed>
-
-<h1>
-    <a href="https://github.com/bumble-tech/bumble-doc-gen/blob/master/src/DocGenerator.php#L46">DocGenerator</a> class:
-</h1>
-
+[BumbleDocGen](../README.md) **/**
+DocGenerator
 
+---
 
 
+# [DocGenerator](https://github.com/bumble-tech/bumble-doc-gen/blob/master/src/DocGenerator.php#L46) class:
 
 ```php
 namespace BumbleDocGen;
 
 final class DocGenerator
 ```
+Class for generating documentation.
 
-<blockquote>Class for generating documentation.</blockquote>
-
-
-
-
-
-
-<h2>Initialization methods:</h2>
-
-<ol>
-<li>
-    <a href="#m-construct">__construct</a>
-    </li>
-</ol>
-
-<h2>Methods:</h2>
-
-<ol>
-<li>
-    <a href="#madddocblocks">addDocBlocks</a>
-    - <i>Generate missing docBlocks with LLM for project class methods that are available for documentation</i></li>
-<li>
-    <a href="#maddplugin">addPlugin</a>
-    </li>
-<li>
-    <a href="#mgenerate">generate</a>
-    - <i>Generates documentation using configuration</i></li>
-<li>
-    <a href="#mgeneratereadmetemplate">generateReadmeTemplate</a>
-    - <i>Creates a `README.md` template filled with basic information using LLM</i></li>
-<li>
-    <a href="#mgetconfiguration">getConfiguration</a>
-    </li>
-<li>
-    <a href="#mgetconfigurationkey">getConfigurationKey</a>
-    </li>
-<li>
-    <a href="#mgetconfigurationkeys">getConfigurationKeys</a>
-    </li>
-<li>
-    <a href="#mparseandgetrootentitycollectionsgroup">parseAndGetRootEntityCollectionsGroup</a>
-    </li>
-<li>
-    <a href="#mserve">serve</a>
-    - <i>Serve documentation</i></li>
-</ol>
+## Initialization methods
 
+1. [__construct](#m-construct) 
+## Methods
 
-<h2>Constants:</h2>
-<ul>
-            <li><a name="qlog-file-name"
-               href="#qlog-file-name">#</a>
-            <code>LOG_FILE_NAME</code>                   <b>|</b> <a href="/src/DocGenerator.php#L49">source
-                    code</a> </li>
-            <li><a name="qversion"
-               href="#qversion">#</a>
-            <code>VERSION</code>                   <b>|</b> <a href="/src/DocGenerator.php#L48">source
-                    code</a> </li>
-    </ul>
+1. [addDocBlocks](#madddocblocks) - Generate missing docBlocks with LLM for project class methods that are available for documentation
```

Comment:
It seems to have gotten a bit confused here, everything listed as `1.`

Reply:
This is not an mistake, but the correct syntax for a numbered list :)